src/components/include/hypercall.h

```diff
@@ -0,0 +1,102 @@
+#ifndef HYPERCALL_H
+#define HYPERCALL_H
+
+#include <cos_kernel_api.h>
+/* TODO: using token from sync inv creation */
+
+/*
+ * strong assumption:
+ *  - capid_t though is unsigned long, only assuming it occupies 16bits for now
+ */
+#define HYPERCALL_ERROR 0xFFFF
```

Comment:
Need to remove this. Not used anywhere after modifying to use slots from the calling component.